types/FluentJSONSchema.d.ts

```diff
@@ -50,7 +50,9 @@ type FORMATS = {
   URI: 'uri'
   TIME: 'time'
   DATE: 'date'
-  DATE_TIME: 'date-time'
+  DATE_TIME: 'date-time',
+  ISO_TIME: 'iso-time',
```

Comment:
```suggestion
  DATE_TIME: 'date-time'
  ISO_TIME: 'iso-time'
```
Don't think the trailing commas are needed looking at the rest of it?